package.json

```diff
@@ -49,35 +41,38 @@
       "@babel/plugin-proposal-object-rest-spread"
     ]
   },
-  "browserify": {
-    "transform": [
-      "babelify"
-    ]
-  },
   "devDependencies": {
     "@babel/cli": "^7.2.3",
     "@babel/core": "^7.3.4",
     "@babel/plugin-proposal-object-rest-spread": "^7.3.4",
     "@babel/preset-env": "^7.3.4",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.0.0",
-    "assume": "^2.1.0",
-    "assume-sinon": "^1.0",
+    "assume": "^2.2.0",
+    "assume-sinon": "^1.0.1",
     "babel-eslint": "^10.0.1",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
     "eslint": "^5.14.1",
-    "eslint-config-godaddy-react": "^4.1.0",
+    "eslint-config-godaddy-react": "^5.0.0",
     "eslint-plugin-json": "^1.4.0",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-mocha": "^5.3.0",
     "eslint-plugin-react": "^7.12.4",
-    "mocha": "^6.0.2",
-    "nyc": "^13.3.0",
+    "jsdom": "^15.1.1",
+    "mocha": "^6.1.4",
+    "nyc": "^14.1.1",
     "prop-types": "^15.7.2",
     "react": "^16.3.0",
+    "react-dom": "^16.8.6",
     "react-test-renderer": "^16.3.0",
+    "setup-env": "^1.2.2",
     "sinon": "^7.2.5"
   },
   "peerDependencies": {
     "react": "^16.4.0 || ^17.0.0"
```

Comment:
that's pretty ambitious of it considering there's no way to test it or be sure it's going to work 😂